includes/class-indieauth-authorize.php

```diff
@@ -15,12 +15,56 @@ public function __construct() {
 		add_filter( 'determine_current_user', array( $this, 'determine_current_user' ), 15 );
 		add_filter( 'rest_authentication_errors', array( $this, 'rest_authentication_errors' ) );
 		add_filter( 'rest_index', array( $this, 'register_index' ) );
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
 
 		add_action( 'send_headers', array( $this, 'http_header' ) );
 		add_action( 'wp_head', array( $this, 'html_header' ) );
 
 		add_filter( 'indieauth_scopes', array( $this, 'get_indieauth_scopes' ), 9 );
 		add_filter( 'indieauth_response', array( $this, 'get_indieauth_response' ), 9 );
+
+	}
+
+	// Test Route for Authorization
+	public static function register_routes() {
```

Comment:
I wouldn't mix the test endpoint with the production system. I would encapsulate this in a separate class, so it is possible to enable/disable it via WP_DEBUG for example